rllib/utils/metrics/tests/test_metrics_logger.py

```diff
@@ -12,7 +12,7 @@ def logger():
     return MetricsLogger(root=True)
 
 
-def test_basic_value_logging(logger):
+def test_log_value(logger):
```

Comment:
Just some renaming of test methods.
There is no "test_advanced_log_value" test or so.